src/offline/mod_offline_rdbms.erl

```diff
@@ -34,22 +34,91 @@
          remove_old_messages/2,
          remove_user/2]).
 
+-import(mongoose_rdbms, [prepare/4, execute_successfully/3]).
+
 -include("mongoose.hrl").
 -include("jlib.hrl").
 -include("mod_offline.hrl").
 
 -define(OFFLINE_TABLE_LOCK_THRESHOLD, 1000).
 
 init(_Host, _Opts) ->
+    prepare_queries(),
+    ok.
+
+prepare_queries() ->
+    prepare(offline_insert, offline_message,
+            [username, server, from_jid, timestamp, expire,
+             packet, permanent_fields],
+            <<"INSERT INTO offline_message "
+              "(username, server, from_jid, timestamp, expire,"
+              " packet, permanent_fields) "
```

Comment:
Why is the order changed here? It was different before and this one is also different from the one in the SQL schema...